src/topology/continuous_function/compact.lean

```diff
@@ -394,4 +422,27 @@ end
 
 end comp_right
 
+section weierstrass
+variables {X : Type*} [topological_space X] [t2_space X] [locally_compact_space X]
+variables {E : Type*} [normed_group E] [complete_space E]
+
+/-- **Weierstrass M-test** -/
```

Comment:
Even though I expect this lemma will almost always be applied to sequences of continuous functions, I think we should add some words indicating it's only a special case. Maybe just:
```suggestion
/-- **Weierstrass M-test** for continuous functions -/
```
I'm also not sure this is the right file for this lemma since the rest of it is about functions on compact spaces. 

Maybe a new file `weierstrass-test.lean`?

Reply:
In fact, now I think of it, this is not really the Weierstrass M-test, because I think that usually refers to the result (for a compact space) that `summable (λ i, ∥F i∥)` implies `summable F`.  (We don't need this result explicitly in mathlib because it's a special case of `summable_of_summable_norm`.  Do you happen to have another naming idea?

I can move it to a new file if you like, but personally I don't think it's so out of place here ...

Reply:
> Do you happen to have another naming idea?
I think the Lean name `continuous_map.summable_of_locally_summable_norm` is fine. How about just dropping the doc string altogether?

> personally I don't think it's so out of place here
OK then, it's certainly not a crazy place for it.

Reply:
```suggestion
```